kreuzberg/_extractors/_email.py

```diff
@@ -111,20 +110,83 @@ def _extract_email_body(self, parsed_email: dict[str, Any], text_parts: list[str
                 h.ignore_links = True
                 h.ignore_images = True
                 converted_text = h.handle(html_content)
-                text_parts.append(f"\n{converted_text}")
+                text_parts.append(converted_text)
             else:
-                clean_html = _HTML_TAG_PATTERN.sub("", html_content)
+                cleaned = re.sub(r"<script[^>]*>.*?</script>", "", html_content, flags=re.IGNORECASE | re.DOTALL)
+                cleaned = re.sub(r"<style[^>]*>.*?</style>", "", cleaned, flags=re.IGNORECASE | re.DOTALL)
+                clean_html = _HTML_TAG_PATTERN.sub("", cleaned)
                 clean_html = unescape(clean_html)
-                text_parts.append(f"\n{clean_html}")
+                clean_html = (
+                    clean_html.replace("\u201c", '"')
```

Comment:
Create a precompiled regex constant at the top of the file and use it instead of these replaces